packages/components/src/Accordion/AccordionItem.tsx

```diff
@@ -93,14 +93,16 @@ export const AccordionItem = ({
           )}
         </button>
       </FocusRing>
-      {expanded || defaultExpanded ? (
-        <div
-          {...mergeProps(regionProps, focusProps, stateProps)}
-          className={classNames.item}
-        >
-          {item.props.children}
-        </div>
-      ) : null}
+      <div
+        {...mergeProps(regionProps, focusProps, stateProps)}
+        className={
+          expanded || defaultExpanded
+            ? classNames.item
+            : cn(classNames.item, classNames.hideItem)
```

Comment:
for me it looks a bit strange to have an `item` and `hideItem` I would put the `hidden ` in the file 🤔

Reply:
There should be aria/data attributes for that (https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded)

Reply:
@sarahgm I thought we try to avoid to write directly styles in the component files?But I agree to not use hideItem. Maybe we can merge it.

@sebald Is there a way to say "When the button has aria-expanded=false hide all siblings/items?".